ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -120,8 +120,14 @@ - (RTCVideoTrack *)createVideoTrack:(NSDictionary *)constraints {
     NSMutableArray *devices = [NSMutableArray array];
     NSArray *videoDevices = [AVCaptureDevice devicesWithMediaType:AVMediaTypeVideo];
     for (AVCaptureDevice *device in videoDevices) {
+        NSString *position = @"";
+        if (device.position == 1) {
```

Comment:
Please don't use numbers, there are defined constants.

Reply:
Ok, sorry. Can you check it now?